policy/policy_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+import (
+	"errors"
+	"io/ioutil"
+	"reflect"
+	"testing"
+
+	sce "github.com/ossf/scorecard/v2/errors"
+)
+
+//nolint
```

Comment:
Let's not add `nolint` which disables all linters. Ideally would be great to fix the failing lint issue, but if it isn't worth taking the effort to fix, specify the linter explicitly - `// nolint: <linterName>`.